api/rest/property_value.md

```diff
@@ -6,156 +6,492 @@ parent: REST
 
-```bash
-curl -X POST 'https://api.datacommons.org/node/property-values' \
--d '{"dcids": ["geoId/05", "geoId/06"], "property": "name"}'
-```
+When actually putting together your request, you can choose from two options. If you intend to use only a small number of parameters, you may want to use the simpler formatting offered by the GET method, which makes requests against the main endpoint while altering the query parameters incorporated into the URL. For more complex queries, or if you prefer to utilize a static URL, a POST request likely makes more sense. To use it, make a POST request against the main endpoint while changing the fields of the JSON body it sends.
 
-<iframe width="100%" height="300" src="//jsfiddle.net/datacommonsorg/285dcqb4/12/embedded/" allowfullscreen="allowfullscreen" allowpaymentrequest frameborder="0"></iframe>
+## What to expect in the response
 
-## Success Response
-
-### **Code**: `200 OK`
-
-**Response content example**
+Your response will always look like this:
 
 ```json
 {
     "payload": "<payload string>",
 }
 ```
 
-The equivalent JSON of "payload string" is as follows.
+Here `"<payload string>"` is replaced by JSON, whose structure changes depending on whether the response contains node references.
 
-**Example 1:** property values that are not nodes.
+**Structure 1:** Response for property values that are not node references.
 
 ```json
 {
-    "geoId/05": {
-        "out": [
-          {
-            "value": "Arkansas",
-            "provenanceId": "dc/sm3m2w3"
-          }
-        ]
-    },
-    "geoId/06": {
-        "out": [
+    "dcid": {
+        "direction": [
           {
-            "value": "California",
-            "provenanceId": "dc/sm3m2w3"
-          }
+            "value": "string",
+            "provenanceId": "string"
+          },
+          ...
         ]
     },
+    ...
 }
 ```
 
-**Example 2:** property values that are node references.
+**Structure 2:** Response for property values that are node references.
 
 ```json
 {
-    "geoId/06":
+    "dcid":
         {
-        "in": [
+        "direction": [
             {
-              "dcid": "ipedsId/486619",
-              "name": "Academy of Professional Cosmetology",
-              "provenanceId": "dc/89fk9x3",
-              "types": ["CollegeOrUniversity"]
+              "dcid": "string",
+              "name": "string",
+              "provenanceId": "string",
+              "types": "string[]"
             },
-            {
-              "dcid": "ipedsId/485731",
-              "name": "The Beauty School",
-              "provenanceId": "dc/89fk9x3",
-              "types": ["CollegeOrUniversity"]
-            }
+            ...
         ]
+    },
+    ...
+}
+```
+
+**NOTES:** 
+ - The `provenanceId` is the DCID of the provenance for the corresponding value.
+ - You can run `JSON.parse()` on the `payload` field to retrieve the data. For example, in JavaScript: `var data = JSON.parse(response['payload'])`.
```

Comment:
Should we move up these notes, so it is clear how to get to the queried value?

Reply:
I merged these notes into the text.